QuickOSM/quick_osm.py

```diff
@@ -159,8 +167,35 @@ def open_dialog(self):
         """Create and open the main dialog."""
         from QuickOSM.ui.dialog import Dialog
         dialog = Dialog()
+        self.open_copyright_message(dialog)
         dialog.exec_()
 
+    @staticmethod
+    def open_copyright_message(dialog: QDialog):
+
+        def read_copyright():
+            open_webpage('https://www.openstreetmap.org/copyright')
```

Comment:
You could make the link clickable in the text I mean, not a button.
Use `<a href></a>`.